gaphor/UML/classes/association.py

```diff
@@ -420,24 +404,25 @@ def max_text_size(size1, size2):
 
         if abs_rc > 6:
             # horizontal line
-            name_dy = -ofs * 2 - name_h
-            mult_dy = ofs * 2
+            name_dy = -padding_top - padding_bottom - name_h
```

Comment:
This is much more readable :+1:

Reply:
And you can change the distance from CSS (although it doesn't take into account the angle of the line end.